texthero/preprocessing.py

```diff
@@ -17,6 +17,34 @@
 
 from typing import List, Callable
 
+# REGEX pattern constants
+PATTERN_REMOVE_DIGITS_BLOCK = r"\b\d+\b"
```

Comment:
Probably, the words `REMOVE`, `REPLACE` and `PATTERN_` are not necessary:

PATTERN_REMOVE_DIGITS_BLOCK -> DIGITS_BLOCK
PATERN_REMOVE_CURLY_BRACKETS -> CURLY_BRACKETS
...